src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -255,18 +257,18 @@ public BlockPos getPositionOfChestWithTool(@NotNull final String tool,final int
         if(building != null)
         {
             if((minLevel != -1
-                    && InventoryUtils.isPickaxeInTileEntity(building.getTileEntity(), minLevel, requestingBuilding.getBuildingLevel()))
-                    || InventoryUtils.isToolInTileEntity(building.getTileEntity(), tool, requestingBuilding.getBuildingLevel()))
+                    && InventoryUtils.isPickaxeInTileEntity(building.getTileItemHandler(), minLevel, requestingBuilding.getBuildingLevel()))
+                    || InventoryUtils.isToolInTileEntity(building.getTileItemHandler(), tool, requestingBuilding.getBuildingLevel()))
             {
                 return building.getLocation();
             }
 
-            for(@NotNull final BlockPos pos : building.getAdditionalCountainers())
+            for(@NotNull final BlockPos pos : building.getAdditionalContainers())
             {
                 final TileEntity entity = worldObj.getTileEntity(pos);
                 if (entity instanceof TileEntityChest
-                        && ((minLevel != -1 && InventoryUtils.isPickaxeInTileEntity((TileEntityChest) entity, minLevel, requestingBuilding.getBuildingLevel()))
-                        || InventoryUtils.isToolInTileEntity((TileEntityChest) entity, tool, requestingBuilding.getBuildingLevel())))
+                        && ((minLevel != -1 && InventoryUtils.isPickaxeInTileEntity(entity.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, null), minLevel, requestingBuilding.getBuildingLevel()))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 208 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)